src/cloud/trouble/error-codes.md

```diff
@@ -0,0 +1,194 @@
+---
+group: cloud-guide
+title: Ece-tools error codes
+---
+
+This page helps you to detect the problem in case of failed deployment process.
+
+
+List of possible errors:
+
+### Build stage
+
+| Step | Description | Error code |
+| --- | --- | --- |
+| general | File `./app/etc/env.php` is not writable | [2](#ENV_PHP_IS_NOT_WRITABLE) |
+| | Configuration isn't defined in schema.yaml | [3](#CONFIG_NOT_DEFINED) |
+| | Failed to parse .magento.env.yaml | 4 |
```

Comment:
won't it be suggestion to check that values in this files are in correct format?

Reply:
I see that you added suggestion, but need to link it here ##### CONFIG_PARSE_FAILED